modules/pages/partials/travel-app-data-model.adoc

```diff
@@ -39,14 +39,13 @@ For airline documents, the value of the `type` field is `airline`.
 ----
 airline_24 			   ←This is the key, which also acts as a primary key
 {
-  "active": "Y",
-  "callsign": "AMERICAN",
-  "country": "United States",
+  "id": 24,
+  "type": "airline",               ←This is the type identifier for the document
```

Comment:
do we still use the type identifier at all now that we have collections?

Reply:
Well, it's still part of the documents it seems, and I think it might still be required for creating indexes? Not 100% sure, but it's definitely still included in the documents.